.github/workflows/ash-build-and-scan.yml

```diff
@@ -69,7 +69,7 @@ jobs:
           set +e
 
           # Run ASH against itself
-          ./ash --source-dir $(pwd) --output-dir ash_output | \
+          ./ash --source-dir $(pwd) --output-dir ash_output --debug | \
```

Comment:
@rafaelpereyra I added it explicitly since this pipeline primarily serves us in the repo, as it is technically another function that we want to ensure doesn't break anything if passed in. Plus it allows us to introspect via pipeline logs during PRs a bit deeper.